LinkedList/Queue.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Queue.h"
+#include <iostream>
+
+
+TYPE Queue::Front() const
+{
+	if (!MyLinkedList::isHeadEmpty()) 

```

Comment:
U can make "IsEmpty" public function and then change "Front" to "return head->value"